app/frontend/src/components/JWTDebug.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <v-bottom-sheet v-if="devMode" inset v-model="sheet">
```

Comment:
I would put the v-if="devMode" up a component at the point where this component is added (Header.vue) so someone looking at the higher level layout knows when this component is rendered or not.

Reply:
Had an internal discussion about this - leaving this for now as this is the only component checking for development mode status. If we need more devMode things, then we will need to consider either a slot pattern (similar to authenticated) or moving the v-if logic up the component chain.